hbase-rest/src/main/java/org/apache/hadoop/hbase/rest/model/ScannerModel.java

```diff
@@ -456,6 +518,12 @@ public FilterModel(Filter filter) {
           this.filters = new ArrayList<>();
           this.filters.add(new FilterModel(((WhileMatchFilter) filter).getFilter()));
           break;
+        case FuzzyRowFilter:
+          this.fuzzyKeys = new ArrayList<>();
```

Comment:
nit: it might be worth to extract the size of the fuzzyKeys and initialize the array list with that as you do in other cases.